repairnator/repairnator-core/src/main/java/fr/inria/spirals/repairnator/LauncherUtils.java

```diff
@@ -113,30 +112,21 @@ public static FlaggedOption defineArgInput(String helpMessage) {
         FlaggedOption opt = new FlaggedOption("input");
         opt.setShortFlag('i');
         opt.setLongFlag("input");
-        opt.setStringParser(FileStringParser.getParser().setMustExist(true).setMustBeFile(true));
-        opt.setRequired(true);
+        opt.setDefault("/tmp/input.txt");
         opt.setHelp(helpMessage);
         return opt;
     }
 
     public static File getArgInput(JSAPResult arguments) {
-        return arguments.getFile("input");
+        return arguments.getObject("input") instanceof File ? (File) arguments.getObject("input") : new File(arguments.getString("input"));
     }
 
     public static FlaggedOption defineArgOutput(LauncherType launcherType, String helpMessage) {
         FlaggedOption opt = new FlaggedOption("output");
         opt.setShortFlag('o');
         opt.setLongFlag("output");
-
-        FileStringParser fileStringParser = FileStringParser.getParser();
-        if (launcherType == LauncherType.SCANNER || launcherType == LauncherType.CHECKBRANCHES) {
-            fileStringParser.setMustBeFile(true);
-        } else {
-            fileStringParser.setMustBeDirectory(true).setMustExist(true);
-        }
-        opt.setStringParser(fileStringParser);
-
-        if (launcherType == LauncherType.DOCKERPOOL || launcherType == LauncherType.REALTIME || launcherType == LauncherType.CHECKBRANCHES) {
+        opt.setDefault("/tmp/repairnator-output");
```

Comment:
I am not sure exactly what this change would do, today when we run the RTScanner logs are created in the `$HOME` directory of the user running the RTScanner, and output is not to a single file, but rather there is log files for each container run as well as for the RTScanner itself. As I understand these changes, they would not allow this behaviour?

Reply:
Realised this might be mitigated by the changes below, but I will keep this comment for now.